cli/internal/destinations/postgresql/README.md

```diff
@@ -0,0 +1,20 @@
+# CloudQuery PostgreSQL Destination Plugin
+
+This destination plugin let's you sync data from any CloudQuery source to any PostgreSQL compatible database.
+
+Known supported databases:
+
+- PostgreSQL > v10
+
+## PostgreSQL Spec
```

Comment:
```suggestion
## Configuration
```

Can we use configuration instead of `spec`?

Reply:
I think we need to be consistent with the configuration as we use `spec` (we went the k8s way so should be familiar to some users) so would rather leave as is.